File: L-69.c
#include<stdio.h>
#include<string.h>
#pragma warning(disable:4996)


int main()
{
	int num[6],d,f=0;

	for (size_t i = 0; i < 6; i++)
	{
		scanf("%d", &num[i]);
	}
	for (size_t i = 0; i < 4; i++)
	{
		for (size_t j = i+1; j < 4; j++)
		{
			int di =(num[i] - num[j]);
            if (di<0)
            {
                di = -di;
            }
			if (di>num[5])
			{
				int p;
				if (num[i]>num[j])
				{
					p = j;
				}
				else
				{
					p = i;
				}
				if (f>0 && d==p)
				{

				}
				else
				{
					d = p;
					f++;
				}
			}
		}
	}
	for (size_t i = 0; i < 4; i++)
	{
		if (num[i] < num[4])
		{
			if (f==1 && d==i)
			{

			}
			else
			{
				d = i;
				f++;
			}
		}
	}
	if (f==0)
	{
		printf("Normal\n");
	}
	else if (f==1)
	{
		printf("Warning: please check #%d!\n", d+1);
	}
	else
	{
		printf("Warning: please check all the tires!\n");
	}
	return 0;
}

File: L-69.py
li=list(map(int,input().split()))


d=0
c=0
dd=[]
cc=[]

for i in range(4):
    if li[i]<li[4]:
        d+=1
        dd.append(i)

for i in range(4):
    for j in range(i,4):
        if abs(li[i]-li[j])>li[5]:
            x=min([li[i],li[j]])
            cc.append(li.index(x))
cc=set(cc)
c=len(cc)
cc=list(cc)
print(cc)
if c+d==0:
    print('Normal')
elif c+d==1:
    if c==1:
        t=cc
    else:
        t=dd
    print('Warning: please check #%d!'%(t[0]+1))
else:
    print('Warning: please check all the tires!')